htdocs/lib/MapMarkers.js

```diff
@@ -410,10 +413,13 @@ AprsMarker.prototype.draw = function() {
         div.style.opacity = null;
     }
 
-    var point = this.getProjection().fromLatLngToDivPixel(this.position);
-    if (point) {
-        div.style.left = point.x - 12 + 'px';
-        div.style.top = point.y - 12 + 'px';
+    // AF: this is for GMaps
```

Comment:
same as above

Reply:
The APRSMarker fix is needed for leaflet, because I'm using the rendering part (creating and updating of the divs)